### PR TITLE
Adding  pre-commit hook with eslint checking and prettier format And testing it

### PR DESCRIPTION
**Pull request by MohammadSommakia**

### CODE REVIEW

File: pages/a.js.rej [hunk 1]
```diff
@@ -0,0 +1,4 @@
+diff a/pages/a.js b/pages/a.js	(rejected hunks)
```
Comment (NourSammour) [inline pages/a.js.rej]: What is this?

File: stories/index.js.rej [hunk 2]
```diff
@@ -0,0 +1,5 @@
+diff a/stories/index.js b/stories/index.js	(rejected hunks)
```
Comment (NourSammour) [inline stories/index.js.rej]: What is this?

File: stories/index.js [hunk 3]
```diff
@@ -4,35 +4,38 @@ import { Button } from '@storybook/react/demo';
...
-margin: 0 1em;
-padding: 0.25em 1em;
-${props =>
-  props.primary &&
-  css`
-    background: #03a9f455;
-    border: 0px;
-    color: #FAFAFA;
-  `};
+  background: #fafafabb;
```
Comment (NourSammour) [inline stories/index.js:7]: Why you did changes here?

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Comment by MohammadSommakia** [comment]
i did not any changes eccept package.json

File: pages/a.js.rej (hunk 1 above)
Comment (MohammadSommakia) [reply pages/a.js.rej]: it is generated files i will delete them then i will push

**Pull request closed without merging**